src/redux/App/reducer.js

```diff
@@ -31,9 +31,6 @@ const initState = new Map({
 
 export default function appReducer(state = initState, action) {
   switch (action.type) {
-    case actions.SET_APP_LOCATION: {
```

Comment:
also dont forget to remove the default `appLocation`

Reply:
fixed